.github/workflows/auto-approve.yml

```
@@ -0,0 +1,13 @@
+name: Auto Approve Dependabot PRs
+
+on:
+  pull_request
+
+jobs:
+  auto-approve:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hmarr/auto-approve-action@v2.0.0
+      if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
+      with:
+        github-token: "${{ secrets.ORIGAMI_FOX_ACCESS_TOKEN }}"
```

o-permutive is not maintained by the Origami team. I wonder if the default github token will work?
```suggestion
        github-token: "${{ secrets.GITHUB_TOKEN }}"
```